datadog_checks_dev/tests/tooling/configuration/consumers/model/test_duplicate_hidden.py

```diff
@@ -0,0 +1,119 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from ...utils import get_model_consumer, normalize_yaml
+
+pytestmark = [pytest.mark.conf, pytest.mark.conf_consumer, pytest.mark.conf_consumer_model]
+
+
+def test():
+    consumer = get_model_consumer(
+        """
+        name: test
+        version: 0.0.0
+        files:
+        - name: test.yaml
+          options:
+          - template: instances
+            options:
+            - name: foo
+              required: true
+              description: words
+              value:
+                type: string
+            - name: password
+              description: words
+              value:
+                example: secret
+                type: string
+            - template: instances/http.password
+              overrides:
+                hidden: true
```

Comment:
I have a question. Is this usage legitimate at all? Wouldn't it be better and cleaner to override the description/example/priority of the existing password field rather than defining a new one and hiding the other?

Reply:
The problem is there are multiple:

https://github.com/DataDog/integrations-core/blob/80a8b7eed8c62d93d0bfffaf56f1fc7c6970383e/sonarqube/assets/configuration/spec.yaml#L87-L95